### PR TITLE
Feat/siwe stable

### PR DESCRIPTION
**Pull request by 0xtiki**
siwe without web3modal working locally

### CODE REVIEW

File: hooks/useServer.ts [hunk 1]
```diff
@@ -18,21 +18,294 @@ const MVP_ALLOWED_CHANNELS = {
   "govbot-testing": "959216625180098590"
 };
 
+const channels_response = {
```
Comment (wpmonty) [inline hooks/useServer.ts]: bump these two arrays of example data out to a fixtures.ts  file

Comment (0xtiki) [reply hooks/useServer.ts]: just saw it now. Probably can get rid of them entirely

File: pages/index.tsx [hunk 2]
```diff
@@ -365,16 +394,4 @@ const Home: NextPage = waitlistDisabled => {
   )
 }
 
-export const getServerSideProps: GetServerSideProps = async context => {
-  const waitlistDisabled =
-    context.query.waitlist === 'true'
-      ? true
-      : process.env.NEXT_PUBLIC_WAITLIST_DISABLED === 'true'
```
Comment (wpmonty) [inline pages/index.tsx:372]: why are we removing waitlist override?

File: pages/_app.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ function MyApp({ Component, pageProps: { session, ...pageProps } }: AppProps) {
   return (
     <SessionProvider session={session}>
       <ChakraProvider theme={theme}>
-        <NavBar waitlistDisabled={pageProps.waitlistDisabled ?? false} />
+        <NavBar />
```
Comment (wpmonty) [inline pages/_app.tsx]: need to support waitlist

File: package.json [hunk 4]
```diff
@@ -25,14 +25,15 @@
...
+    "ethers": "^5.5.1",
     "framer-motion": "^5.6.0",
     "fuse.js": "^6.5.3",
     "jotai": "^1.6.1",
     "lodash": "^4.17.21",
     "luxon": "^2.3.1",
     "moment": "^2.29.2",
     "next": "12.1.0",
-    "next-auth": "^4.10.3",
+    "next-auth": "^4.3.2",
```
Comment (wpmonty) [inline package.json:36]: reduced next-auth version?

File: hooks/useServer.ts (hunk 1 above)
Comment (wpmonty) [reply hooks/useServer.ts]: just break out for now, replace in new PR

File: components/polls/PollForm.tsx [hunk 5]
```diff
@@ -319,10 +376,104 @@ const PollForm: React.FC<BoxProps> = ({ ...props }) => {
...
+                    )
+                    clearErrors('token_strategies');
+                    (i?.label === EMPTY_STRATEGY_NAME) ? setIsTokenVote(false) : setIsTokenVote(true);
+                  }}
+                />
+              )}
+            />
+          </FormControl>
+
+          <FormControl isInvalid={!!errors.block_height?.message}>
```
Comment (wpmonty) [inline components/polls/PollForm.tsx:419]: Hide this field if "Bankless Dao" strategy is not set (the logic should really be "if not token-based strategy".

File: components/polls/PollForm.tsx [hunk 6]
```diff
@@ -8,24 +8,32 @@ import {
...
-import { useEffect } from 'react'
+import {useEffect, useState} from 'react'
 import useServer from 'hooks/useServer'
+import useStrategies from 'hooks/useStrategies'
 import PollOption from './PollOption'
+import {useAtom} from "jotai";
+import {userAtom} from "../../atoms";
 
-interface Poll {
+const EMPTY_STRATEGY_NAME = 'Empty strategy (returns voting power of 1)';
```
Comment (wpmonty) [inline components/polls/PollForm.tsx]: rename "Standard (1 Vote = 1 Vote)"

File: components/polls/PollForm.tsx [hunk 7]
```diff
@@ -319,10 +376,104 @@ const PollForm: React.FC<BoxProps> = ({ ...props }) => {
...
+          {/*  ╚═════╝╚═╝  ╚═╝╚══════╝ ╚═════╝ ╚═════╝ ╚══════╝╚═╝  ╚═╝   ╚═╝   ╚═╝ ╚═════╝ ╚═╝  ╚═══╝╚══════╝
+          {/*                                                                                                  */}
+          <VStack spacing={2} marginTop={6}>
+            <Heading fontSize="3xl" as="h1">Calculations</Heading>
+            <Text fontSize="m">How should votes be tallied?</Text>
+          </VStack>
+
+           <FormControl>
+            <FormLabel mt='1rem' htmlFor='tokenStrategies'>
+              Token Strategy
```
Comment (wpmonty) [inline components/polls/PollForm.tsx]: rename label "Voting Strategy"

File: components/polls/PollForm.tsx [hunk 8]
```diff
@@ -319,10 +376,104 @@ const PollForm: React.FC<BoxProps> = ({ ...props }) => {
...
+            </FormLabel>
+            <Controller
```
Comment (wpmonty) [inline components/polls/PollForm.tsx:393]: wherever "Bankless Dao" is rename to "BANK - Token Weighted"

File: components/polls/PollForm.tsx (hunk 5 above)
Comment (wpmonty) [reply components/polls/PollForm.tsx:419]: so we might need a "strategy type" prop in the future, for now we can hard-code

File: components/RouteGuard.tsx [hunk 9]
```diff
@@ -13,16 +13,14 @@ const RouteGuard: React.FC = ({ children }) => {
   const { status } = useSession()
   const router = useRouter()
 
-  const url = router.query.waitlist ? '/?waitlist=true' : '/'
-
   useEffect(() => {
     if (status === 'unauthenticated') {
-      router.push(url)
+      router.push({ pathname: '/' })
```
Comment (wpmonty) [inline components/RouteGuard.tsx]: revert these to implement waitlist

**Pull request merged**